internal/transform/transform.go

```diff
@@ -453,3 +454,22 @@ func walk(doc *astro.Node, cb func(*astro.Node)) {
 	}
 	f(doc)
 }
+
+// This function merges the values of `class=""` and `class:list=""` in `class:list`
+func mergeClassList(doc *astro.Node, n *astro.Node, opts *TransformOptions) {
+	var classListAttrValue string
+	var classListAttrIndex int
+	var classAttrValue string
+	for i, a := range n.Attr {
+		if a.Key == "class:list" {
+			classListAttrValue = a.Val
+			classListAttrIndex = i
+		}
+		if a.Key == "class" {
+			classAttrValue = a.Val
+		}
+	}
+	if classListAttrIndex > 0 {
+		n.Attr[classListAttrIndex].Val = fmt.Sprintf("['%s', %s]", classAttrValue, classListAttrValue)
```

Comment:
It looks like if you have `class:list` without `class` it will still merge class, maybe as an empty string? Would be nice to avoid that.

Reply:
Good catch, thank you